src/libecalc/dto/result/tabular_time_series.py

```diff
@@ -23,7 +30,22 @@ def to_dataframe(
 
         for attribute_name, attribute_value in self.__dict__.items():
             if isinstance(attribute_value, TimeSeries):
-                column_name = f"{attribute_name}[{attribute_value.unit}]"
+                unit_value = attribute_value.unit
+                if isinstance(attribute_value, TimeSeriesRate):
+                    if attribute_value.rate_type == RateType.STREAM_DAY:
```

Comment:
Start with an if to figure out "sd" vs "cd", then use that variable to set for megawatt vs not mega? One less indent that way